jans-cedarling/cedarling/src/authz/authorize_result.rs

```diff
@@ -5,9 +5,8 @@ use cedar_policy::Decision;
 pub struct AuthorizeResult {
     /// Result of authorization where principal is `Jans::Workload`
     pub workload: cedar_policy::Response,
-    // The response for the user entity
-    // next field will be added in further PR
-    // pub person: cedar_policy::Response,
+    /// Result of authorization where principal is `Jans::User`
+    pub person: cedar_policy::Response,
```

Comment:
why is the principal `Jans::User` and it's named `person`? can't we name this `user` as well so it's less confusing? We could also change `Jans::User` to be `Jans::Person`, im not sure which one is better.

though honestly, it's a ambiguous as well on other parts of the codebase... maybe we need to create a separate issue to resolve the ambiguity later. feel free to resolve this comment.

Reply:
In documentation, wrote that we authorize `person` so that was the point

Reply:
I would like to decide a better name for this field in one of next issue...